src/util/api/likernft/book/index.ts

```diff
@@ -9,15 +9,23 @@ import {
   likeNFTBookCollection,
 } from '../../../firebase';
 import { LIKER_NFT_TARGET_ADDRESS, LIKER_NFT_BOOK_GLOBAL_READONLY_MODERATOR_ADDRESSES } from '../../../../../config/config';
-import { FIRESTORE_BATCH_SIZE, NFT_BOOKSTORE_HOSTNAME } from '../../../../constant';
-import { getNFTsByClassId } from '../../../cosmos/nft';
+import { FIRESTORE_BATCH_SIZE, LIKER_LAND_HOSTNAME, NFT_BOOKSTORE_HOSTNAME } from '../../../../constant';
+import {
+  getISCNFromNFTClassId, getNFTClassDataById, getNFTISCNData, getNFTsByClassId,
+} from '../../../cosmos/nft';
 import { getClient } from '../../../cosmos/tx';
 import { sleep } from '../../../misc';
+import stripe from '../../../stripe';
+import { parseImageURLFromMetadata } from '../metadata';
 
 export const MIN_BOOK_PRICE_DECIMAL = 90; // 0.90 USD
 export const NFT_BOOK_TEXT_LOCALES = ['en', 'zh'];
 export const NFT_BOOK_TEXT_DEFAULT_LOCALE = NFT_BOOK_TEXT_LOCALES[0];
 
+export function getLocalizedTextWithFallback(field, locale) {
+  return field[locale] || field[NFT_BOOK_TEXT_DEFAULT_LOCALE] || undefined;
```

Comment:
Use empty string for fallback?
```suggestion
  return field[locale] || field[NFT_BOOK_TEXT_DEFAULT_LOCALE] || '';
```